Godeps/_workspace/src/k8s.io/kubernetes/pkg/apiserver/apiserver.go

```
@@ -177,7 +177,11 @@ func serviceErrorHandler(requestResolver *APIRequestInfoResolver, apiVersions []
 		}
 	}
 
-	errorJSON(apierrors.NewGenericServerResponse(serviceErr.Code, "", "", "", "", 0, false), codec, response.ResponseWriter)
+	resourceOrKind := requestInfo.Kind
+	if len(resourceOrKind) == 0 {
+		resourceOrKind = requestInfo.Resource
```

agree... just drop the error swallowing for now and open a follow up